From 41ae689ad428568a22f06fcb248f20af52770afe Mon Sep 17 00:00:00 2001
From: Levin Du <djw@t-chip.com.cn>
Date: Wed, 15 Aug 2018 17:22:37 +0800
Subject: [PATCH] arm64: dts: rockchip: enable usb3 for roc-rk3328-cc board

Enable the USB 3.0 OTG controller and set as static xHCI host controller to
support USB 3.0 HOST.

Signed-off-by: Levin Du <djw@t-chip.com.cn>
---
 arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
index 99d0d9912950..6ba67963eaf5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
@@ -295,3 +295,12 @@
 &usb_host0_ohci {
 	status = "okay";
 };
+
+&usbdrd3 {
+	status = "okay";
+};
+
+&usbdrd_dwc3 {
+	dr_mode = "host";
+	status = "okay";
+};
